api/SharedWorkerGlobalScope.json

```diff
@@ -49,9 +49,7 @@
             }
           ],
           "samsunginternet_android": "mirror",
-          "webview_android": {
-            "version_added": "≤37"
-          },
+          "webview_android": "mirror",
```

Comment:
Interesting case, where previous data was wrong, as Webview very likely doesn't support this, cf. [this comment](https://issues.chromium.org/issues/40290702#comment18) in the bug.